### PR TITLE
[chore]: update docs to reflect verbosity for loggingexporter

### PR DESCRIPTION
**Pull request by sakshi1215**
Signed-off-by: Sakshi Patle <sakshi.patle121@gmail.com>

**Description:** 

Changing docs to deprecate loglevel in favour of verbosity

**Link to tracking Issue:** 

fixes https://github.com/open-telemetry/opentelemetry-collector/issues/6448

**Documentation:** 

Changing docs to deprecate loglevel in favour of verbosity

### CODE REVIEW

**Comment by sakshi1215** [comment]
@mx-psi Please check if the changes have been made correctly.

File: exporter/loggingexporter/README.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ Supported pipeline types: traces, metrics, logs
 
 The following settings are optional:
 
-- `loglevel` (default = `info`): the log level of the logging export
-  (debug|info|warn|error). When set to `debug`, pipeline data is verbosely
-  logged.
```
Comment (mx-psi) [inline exporter/loggingexporter/README.md:19]: Let's keep `loglevel` documented on this list but note that it is deprecated

File: exporter/loggingexporter/README.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ Supported pipeline types: traces, metrics, logs
...
+- `verbosity` (default = `normal`): the verbosity of the logging export
+  (detailed|normal|basic). When set to `detailed`, pipeline data is verbosely
+  logged. Anything above loglevel `info` is mapped to verbosity level `basic`.
```
Comment (mx-psi) [inline exporter/loggingexporter/README.md:19]: I would remove this last sentence, I don't think it's very useful for the end users.

```suggestion
  logged.
```

**Review by mx-psi — COMMENTED** [review]
Thanks for the PR! I left a couple minor comments

**Comment by sakshi1215** [comment]
@mx-psi I didn't find a neat way to mention loglevel is deprecated, I have added a note section to mention about the same.

File: exporter/loggingexporter/README.md [hunk 3]
```diff
@@ -15,7 +15,10 @@ Supported pipeline types: traces, metrics, logs
 The following settings are optional:
 
 - `loglevel` (default = `info`): the log level of the logging export
-  (debug|info|warn|error). When set to `debug`, pipeline data is verbosely
+  (debug|info|warn|error). When set to `debug`, pipeline data is verbosely
```
Comment (mx-psi) [inline exporter/loggingexporter/README.md:18]: > @mx-psi I didn't find a neat way to mention loglevel is deprecated, I have added a note section to mention about the same.

What about something like this?

```suggestion
  (debug|info|warn|error). When set to `debug`, pipeline data is verbosely 
      - **Note**: This option has been deprecated in favor of `verbosity`
```

I think it's important that the note is as close as possible to the setting description

**Review by bogdandrutu — APPROVED** [review]

**Review by mx-psi — APPROVED** [review]

**Pull request merged**